mobify/progressive-web-scaffold
# Update onboarding to move to the web folder

Remove the onboarding react app from the native folder and add a container to hold onboarding inside the `web` folder.

 **JIRA**: https://mobify.atlassian.net/browse/HYB-1002

## Changes
- Remove old onboarding
- Add onboarding to web folder

## How to test-drive this PR
- Run `npm run prod:build`
- Run scaffold app in Xcode
- See onboarding
